MAINTAINERS.md

```diff
@@ -5,19 +5,19 @@ for the various backends supported to allow oneDPL to run on CPU, GPU, and FPGA
 which a developer is a lead maintainer are listed below.
 
 
-| Name                  | Github ID           | API Area                                   |
-| --------------------- | ------------------- | ------------------------------------------ |
-| Ruslan Arutyunyan     | @rarutyun           | PSTL Offload                               |
-| Konstantin Boyarinov  | @kboyarinov         | PSTL Offload                               |
-| Pavel Dyakov          | @paveldyakov        | Random number generator APIs               |
-| Mikhail Dvorskiy      | @MikeDvorskiy       | Range-based algorithms, Sort algorithms    |
-| Adam Fidel            | @adamfidel          | Scan algorithms, Experimental Scan KT      |
-| Dan Hoeflinger        | @danhoeflinger      | Histogram, Transform, Iterators            |
-| Alexandr Konovalov    | @Alexandr-konovalov | PSTL Offload                               |
-| Sergey Kopienko       | @SergeyKopienko     | Find algorithms, Experimental Sort KT      |
-| Alexey Kukanov        | @akukanov           | oneDPL architecture, Experimental Sort KT  |
-| Matt Michel           | @mmichel11          | By-segment algorithms, Search algorithms   |
-| Julian Miller         | @julianmi           | Reduce algorithms                          |
-| Dmitriy Sobolev       | @dmitriy-sobolev    | Experimental Sort KT                       |
-| Anuya Welling         | @AnuyaWelling2801   | Experimental Dynamic Device Selection APIs | 
+| Name                  | Github ID           | API Area                                                |
+| --------------------- | ------------------- | ------------------------------------------------------- |
```

Comment:
Formatting

Reply:
Should the final column have a ragged boundary?

Reply:
Done